commit bd54b8bf9c94036024c659e9cf7e0f861a2c188a
Author: Tyler Nijmeh <tylernij@gmail.com>
Date:   Sat May 18 14:38:17 2019 -0700

    fs: Reduce cache pressure
    
    We can more utilize our available RAM better like this.
    
    Signed-off-by: Tyler Nijmeh <tylernij@gmail.com>

diff --git a/fs/dcache.c b/fs/dcache.c
index 452668c9644d..b2bed6cd101b 100644
--- a/fs/dcache.c
+++ b/fs/dcache.c
@@ -71,7 +71,7 @@
  * If no ancestor relationship:
  * arbitrary, since it's serialized on rename_lock
  */
-int sysctl_vfs_cache_pressure __read_mostly = 100;
+int sysctl_vfs_cache_pressure __read_mostly = 50;
 EXPORT_SYMBOL_GPL(sysctl_vfs_cache_pressure);
 
 __cacheline_aligned_in_smp DEFINE_SEQLOCK(rename_lock);
